torchtext/experimental/datasets/raw/text_classification.py

```diff
@@ -161,6 +183,10 @@ def AmazonReviewPolarity(*args, **kwargs):
 
     Arguments:
         root: Directory where the datasets are saved. Default: ".data"
+        data_select: a string or tuple for the returned datasets
+            (Default: ('train', 'test'))
+            By default, both datasets (train, test) are generated. Users could also choose any one or two of them,
+            for example ('train', 'test') or just a string 'train'.
```

Comment:
I don't think the behavior
```
train, = AG_NEWS(data_select='train')
```

is obvious. I think I'd expect a single item to be returned instead of a tuple with a single entry. I'd document this.